README.md

```diff
@@ -1 +1,2 @@
-# code-review
\ No newline at end of file
+# OliverBenard
+Microverse collaboration between Oliver and Benard
```

Comment:
Please write a detailed README.md file. You can use the template provided and customize to your needs. A detailed README file help explain your project and how to use it.